cluster/main.tf

```diff
@@ -0,0 +1,25 @@
+locals {
+  ec2_capacity_provider = var.enable_ec2_capacity_provider ? {
+    ec2_default = {
+      instance_type               = var.ec2_capacity_instance_type
+      security_group_ids          = [var.ec2_capacity_security_group_id]
+      subnet_ids                  = var.ec2_subnet_ids
+      associate_public_ip_address = false
+      min_size                    = var.ec2_capacity_min_size
+      max_size                    = var.ec2_capacity_max_size
+    }
+  } : {}
+}
+
+module "ecs_cluster" {
+  source  = "cloudposse/ecs-cluster/aws"
```

Comment:
This looks like a reference to a local path, which is not committed with these changes (so it would fail);
also if it meant to be a path to a third party, then similarly to @davidkelliott comment, we cannot use it unless it's an already security approved provider